javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinter.java

```diff
@@ -164,27 +170,43 @@ public void concretePropertyChange(Node observedNode, ObservableProperty propert
             LEXICAL_DIFFERENCE_CALCULATOR.calculatePropertyChange(nodeText, observedNode, property, oldValue, newValue);
         }
 
-        private List<TokenTextElement> getMatchingTokenTextElements(JavadocComment oldValue, NodeText nodeText) {
-            JavadocComment javadocComment = oldValue;
-            List<TokenTextElement> matchingTokens = nodeText.getElements().stream()
+        private List<TokenTextElement> getMatchingTokenTextElements(Comment oldValue, NodeText nodeText) {
```

Comment:
should this method name contains the word comment?

Reply:
I renamed the method to "findTokenTextElementForComment" now, that should describe its purpose more accurately